token/token.go

```diff
@@ -23,10 +36,37 @@ var keywords = map[string]Type{
 	"in":     IN,
 }
 
+var dynamic = map[Type]Type{}
+
 // LookupIdent an ident and return a keyword type, or a plain ident
 func LookupIdent(ident string) Type {
 	if tok, ok := keywords[ident]; ok {
 		return tok
 	}
 	return IDENT
 }
+
+// SetTemplatingDelimiters to start and end, or return delimitersLengthError if delimiters length is incorrect
+func SetTemplatingDelimiters(start, end string) error {
+	if len(start) != templateDelimitersLen ||
+		len(end) != templateDelimitersLen {
+		return &delimitersLengthError{[]string{start, end}, templateDelimitersLen}
+	}
+	replace(S_START, Type(start))
```

Comment:
Does the `replace` function really add value in comparison to a simple assignment?

Reply:
agree, fixed